scripts/build.py

```diff
@@ -72,6 +72,10 @@
     'out_dir': os.path.join('build', 'webpack_bundles', '')
 }
 
+# This json file contains a json object. The object's keys are file paths and
+# the values are corresponded hash value. The pathes need to be in posix style,
+# as it is interpreted by the `url-interpolation` service, which assumes the
+# pathes in this file are in posix style.
```

Comment:
pathes --> paths

Also, end with: ... which interprets the paths in this file as URLs. 
(if you think about it, the url interpolation service isn't actually thinking of these as filepaths at all).

Reply:
Updated